operator/pkg/component/component.go

```diff
@@ -27,15 +27,15 @@ import (
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 
 	"istio.io/api/operator/v1alpha1"
+	"istio.io/pkg/log"
+
```

Comment:
nit: remove blank line.

Reply:
This is done by `make format`. I have no control